simp-rake-helpers.gemspec

```diff
@@ -23,28 +23,23 @@ Gem::Specification.new do |s|
   #   for the published gem
   # ensure the gem is built out of versioned files
 
-  s.add_runtime_dependency 'bundler',                   '~> 1.14'
-  s.add_runtime_dependency 'rake',                      '>= 10.0', '< 13.0'
-  s.add_runtime_dependency 'coderay',                   '~> 1.0'
-  s.add_runtime_dependency 'puppet',                    '>= 3.0', '< 6.0'
-  s.add_runtime_dependency 'puppet-lint',               '>= 1.0', '< 3.0'
-  s.add_runtime_dependency 'puppetlabs_spec_helper',    '~> 2.0'
-  s.add_runtime_dependency 'parallel',                  '~> 1.0'
-  s.add_runtime_dependency 'simp-rspec-puppet-facts',   '~> 2.0'
-  s.add_runtime_dependency 'puppet-blacksmith',         '~> 3.3'
-  s.add_runtime_dependency 'simp-beaker-helpers',       '~> 1.0'
-  s.add_runtime_dependency 'parallel_tests',            '~> 2.4'
-  s.add_runtime_dependency 'r10k',                      '~> 2.2'
-  s.add_runtime_dependency 'pager'
-  s.add_runtime_dependency 'rspec',                     '~> 3.0'
-  s.add_runtime_dependency 'beaker',                    '~> 3.14'
-  s.add_runtime_dependency 'beaker-rspec',              '~> 6.1'
-  s.add_runtime_dependency 'rspec-core',                '~> 3.0'
+  s.add_runtime_dependency 'simp-beaker-helpers',     '~> 1.0'
```

Comment:
Should rake helpers pull in beaker helpers?

Reply:
We have both in the Gemfile and it would make it all a bit more modular if you didn't have to install all the beaker gems

Reply:
@jeefberkey It uses beaker for its tests

Reply:
@trevor-vaughan can't you just add it the Gemfile then?

Reply:
@jeefberkey It also uses additions that we added to `simp-beaker-helpers`. No reason to duplicate all of that information.